### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.3][IntelliJ][Wipro]:Focus order is illogical aftert opening the Publish as Run on web app for Containers  window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1447]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app for containers.
4. "Run on web app for Containers" dialog will open.
5. Then hit Tab and observe the focus order.

**Expected Result:**
When "Run on web app for Containers" is opened after hitting 'Tab' keyboard focus should be on the first interactive element in the window.

**Actual Result:**
When "Run on web app for Containers" is opened after hitting 'Tab' keyboard focus is on  "Artifact" combo box in the window.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![focus order is illogical in run on web app containers](https://user-images.githubusercontent.com/22956130/35730046-7561a982-07c5-11e8-9721-09a414b5bfb3.png)

**Comment by Eskibear** [comment]
Cannot reproduce. Refer to the following GIF, pay attention to the focus change.

![ij_tab](https://user-images.githubusercontent.com/2351748/39612374-75583e56-4f91-11e8-94e5-211a1e57a0b9.gif)

**Comment by Eskibear** [comment]
```
IntelliJ IDEA 2017.3.4 (Community Edition)
Build #IC-173.4548.28, built on January 30, 2018
JRE: 1.8.0_152-release-1024-b11 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
```

**Comment by WiproPSAccTest** [comment]
Verified in V3.6.0 Community.
Issue still exists

**Comment by Eskibear** [comment]
a related bug is submitted to IntellIJ, https://youtrack.jetbrains.com/issue/IDEA-191682

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749892
